tfs/reader.py

```diff
@@ -125,25 +144,34 @@ def read_tfs(
     metadata: _TfsMetaData = _read_metadata(tfs_file_path)
```

Comment:
As a reminder: this nicely takes care of reading just enough of the file to determine the headers, the column names and the columns types (plus some other things to optimize reading later on).